lib/ord.js

```diff
@@ -119,7 +119,7 @@ const fGetGroups = (csn, global) => {
  * Hierarchy to check data: cdsrc.json > defaults
  * @returns {Array<object>} The packages array.
  */
-const fGetPackages = (policyLevel,global) => global.env?.packages || (global.aEvents.length) ? defaults.packages(global.namespace,policyLevel,global.capNamespace) : defaults.packages(global.namespace,policyLevel,global.capNamespace).slice(0, 1)
+const fGetPackages = (policyLevel,global) => global.env?.packages || (global.aEvents.length) ? defaults.packages(global.appName,policyLevel,global.capNamespace) : defaults.packages(global.appName,policyLevel,global.capNamespace).slice(0, 1)
```

Comment:
We probably have to add tests to check if the package ORD IDs and partOfPackages in e.g., apiResources still match in different cases after this change.